libbeat/docs/newdashboards.asciidoc

```diff
@@ -126,56 +148,11 @@ If Elasticsearch is running on a different host, then you can use the environmen
 $ ES_URL="http://192.168.3.206:9200" make import-dashboards
 -------------------------------
 
-The command has the following options:
+To see all the available options, check the options below or run:
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-
```

Comment:
I agree that having the output of the help menu is not that useful in the docs, but maybe we show at least somewhere how to get the help menu. What do you think @dedemorton?